lib/rubocop/cop/mixin/assert_refute_same_helper.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    # Provide a method to define offense rule for Minitest cops.
+    module AssertRefuteSameHelper
+      def define_rule(assertion_method)
+        class_eval(<<~RUBY, __FILE__, __LINE__ + 1)
```

Comment:
Since it's specialized for `Minitest/AssertSame` and `Minitest/RefuteSame`, there's likely no need for metaprogramming with eval.

Reply:
But we define different things depending on where we use this method.

Reply:
Updated.